single_asset/README.md

```diff
@@ -58,15 +59,17 @@ $ brew install libsodium libsecp256k1 gmp
 Tests are configured to run on [Flextesa sandbox](https://assets.tqtezos.com/sandbox-quickstart).
 There are two helper scripts in `tezos_mac_tests` module:
 
-* [`start-sandbox.sh`](tezos_mac_tests/start-sandbox.sh) - starts Flextesa sandbox
+* [`start-sandbox.sh`](./tezos_fa2_single_tests/start-sandbox.sh) - starts Flextesa sandbox
 from the docker image
-* [`kill-sandbox.sh`](tezos_mac_tests/kill-sandbox.sh) - kills running Flextesa
+* [`kill-sandbox.sh`](./tezos_fa2_single_tests/kill-sandbox.sh) - kills running Flextesa
 sandbox docker container
 
 ## Installation and Running The Tests
 
 ### Install dependencies
 
+<!-- Aren't those redundant with the above? -->
```

Comment:
Do not remember why exactly this layout. I guess just wanted to have a distinct steps for running the tests